cpp/uuid_module/uuid_module.cpp

```diff
@@ -0,0 +1,40 @@
+#include <uuid/uuid.h>
+
+#include <mg_procedure.h>
+#include <mg_exceptions.hpp>
+#include <mg_utils.hpp>
+
+namespace {
+
+constexpr char const *kFieldUuid = "uuid";
+
+void Generate(const mgp_list *args, const mgp_graph *memgraph_graph, mgp_result *result, mgp_memory *memory) {
+  try {
+    uuid_t id;
+    uuid_generate(id);
+    char *string = new char[100];
+    uuid_unparse(id, string);
+
+    mgp_result_record *record = mgp_result_new_record(result);
+    if (record == nullptr) {
+      throw mg_exception::NotEnoughMemoryException();
+    }
+
+    mg_utility::InsertStringValueResult(record, kFieldUuid, string, memory);
+  } catch (const std::exception &e) {
+    // We must not let any exceptions out of our module.
+    mgp_result_set_error_msg(result, e.what());
+    return;
+  }
```

Comment:
Is the goal of this module to obtain a uuid for each node in graph?

Reply:
The goal is to return a newly generated string UUID just like in [this](https://neo4j.com/labs/apoc/4.1/graph-updates/uuid/#manual-uuids) APOC procedure.

Reply:
Alright that's cool, please write an e2e test before merging this to `main`

Reply:
The e2e test is written and committed, you can try it out.